docs/en/getting-started.md

```diff
@@ -47,6 +47,10 @@ Note that there are two pages:
 
 See the [Developer Platform Developer Guide][3] for details about this architecture.
 
+<div class="alert alert-info">
+You may notice an uncaught <strong>HandshakeTimeoutError</strong> in your JavaScript Console when you interact with the local widget in your browser directly. This is expected. As the <a href="https://github.com/DataDog/apps/blob/master/docs/en/programming-model.md">Developer Platform Developer Guide</a> explains that the Datadog Apps SDK connects to the Datadog User Interface via <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage">postMessage</a> and establishes a communication channel via the <a href="https://github.com/DataDog/apps/tree/master/packages/framepost"><code>@datadog/framepost</code> package</a> and the handshake attempt between the widget and the Datadog UI will timeout when there is no Datadog UI for the SDK has to communicate with.
```

Comment:
I don't think we need to call out that we're using `@datadog/framepost` for iFrame communication, since it's an implementation detail — what do you think?

Reply:
That's probably a good call.

I sort of just did a brain dump.

I'll take a pass at simplifying it.